### ISSUE TITLE
Cannot open settings on macOS

### ISSUE THREAD

**Issue by MatrixRanger98** [xournalpp/xournalpp#5652]
### Operating System

Mac OS

### (Linux Only) Display Server

None

### Installation Method

Nightly build (pre-release)

### Xournal++ Version

1.2.3-dev May 1st

### libgtk Version

3.24.38

### Bug Description

<img width="1744" alt="image" src="https://github.com/xournalpp/xournalpp/assets/48684632/3f3f18df-38ae-4332-ad0f-91857cdd2096">

### Expected Behaviour

Settings should be clickable

### Steps to Reproduce

just install and open it

macOS Sonoma 14.4.1 Apple M3 chip

**Comment by rolandlo** [comment]
We have created an ARM build (of the current development version) for MacOS that can be downloaded at you access it on https://cirrus-ci.com/task/5138787729145856 (under Artifacts). It works much better on Apple M1/M2/M3. We plan to include an official ARM build from the next release version (1.2.4) on.

**Comment by MatrixRanger98** [comment]
I just tried the link but somehow it says the app is damaged.

<img width="260" alt="image" src="https://github.com/xournalpp/xournalpp/assets/48684632/c977d69b-9fe7-4b29-90cb-4534a52c52da">

**Comment by rolandlo** [comment]
You should move it out of quarantine (after you installed it in the `/Applications` folder). From the terminal run:

```shell
xattr -d com.apple.quarantine /Applications/Xournal++.app/
```

**Comment by MatrixRanger98** [comment]
Yeah. It is opened now. But the bug is still there. `Settings...` is grey and can't be clicked. command+, does not work either.

**Comment by rolandlo** [comment]
To access Xournal++'s preferences you should use menu `Edit > Preferences`. The `Xournal++` menu with the `settings` is something we didn't generate ourselves. Do you know any Gtk apps that have this `settings` menu entry working?

**Comment by MatrixRanger98** [comment]
I see. That makes sense.

**Comment by rolandlo** [comment]
After finding some [useful observations](https://github.com/jralls/gtk-osx-build/discussions/93) on how this works on MacOS, I will reopen the issue.

**Comment by Febbe** [comment]
@rolandlo, the reason it does not work with Xournalpp is, that we still use the gtk2 API of menu's. @bhennion is changing this on our way to gtk4, since gtk4 removed all old gtk2 API. So this might be fixed in the next 1.3 release. The last release before we start the development of Xournalpp2.0 will contain it.
